monai/bundle/config_item.py

```diff
@@ -236,7 +236,7 @@ def resolve_module_name(self):
         config = dict(self.get_config())
         target = config.get("_target_")
         if not isinstance(target, str):
-            raise ValueError("must provide a string for the `_target_` of component to instantiate.")
+            return target
```

Comment:
Could you please help add 1 line comment here to explain why return `target` directly if not a string?
I am afraid we may forget about it later.. We already have very complicated logic in the config parser module.

Thanks.